src/Engine/ProtoCore/FFI/CLRFFIFunctionPointer.cs

```diff
@@ -64,6 +64,26 @@ protected FFIMemberInfo(MemberInfo info)
             Info = info;
         }
 
+        internal void CheckForRankReductionAttribute(Dictionary<MethodInfo, Attribute[]> getterAttributes)
```

Comment:
name is okay to me, but just considering others - copyRankReductionAttribute?

Reply:
Would you add some function comments, what you have mentioned in this PR description is good enough

Reply:
@QilongTang yes, I'll add some comments. @mjkkirschner yeah I like that better. Or maybe "AttachRankReductionAttribute"